### PR TITLE
Added Help and Defaults

### PR DESCRIPTION
**Pull request by jfgordon2**
- Add `uni --help` to provide available use options in CLI
- Assume default recipe owner if not provided (since there's currently only one, anyway) for ease of typing -- allows typing just `uni bake yarn-install` for example

image

### CODE REVIEW

File: uni [hunk 1]
```diff
@@ -59,6 +59,34 @@ while [ "$#" -gt 0 ]; do
...
+${STYLE_BOLD}  rm-me, remove-me${RESET_STYLE}            Removes UNI from the system
+
+${STYLE_LIGHT_CYAN}Available Recipes (by Dalton Menezes):${RESET_STYLE}
+${STYLE_YELLOW}  atom-install${RESET_STYLE}                Installs the latest version of Atom Editor
+${STYLE_YELLOW}  telegram-install${RESET_STYLE}            Installs the latest version of Telegram
+${STYLE_YELLOW}  node-install${RESET_STYLE}                Installs the nodejs latest LTS available version and npm
+${STYLE_YELLOW}  yarn-install${RESET_STYLE}                Installs the latest version of Yarn
+${STYLE_YELLOW}  stylishbuntu-theme${RESET_STYLE}          Theme Package
+${STYLE_YELLOW}  nebulosa-theme${RESET_STYLE}              Theme Package
+${STYLE_YELLOW}  mac-theme${RESET_STYLE}                   Theme Package
```
Comment (daltonmenezes) [inline uni]: This shouldn't be hard-coded. Can you fix this bringing this information from GitHub?
In this case, it should fetch from [here](https://github.com/uni-linux/recipes/tree/master/src/daltonmenezes)

**Review by daltonmenezes — CHANGES_REQUESTED** [review]
Thanks @Jamesits , great additions. I made a change request, also can you update the docs?

File: uni (hunk 1 above)
Comment (jfgordon2) [reply uni]: I've updated to grab the README from that github with minor transformation for readability

**Review by daltonmenezes — COMMENTED** [review]
@Jamesits  It will print the following result
img

What if you achieve the following result? It looks cleaner and easier to read / understand. What do you think about it?
img

**Comment by jfgordon2** [comment]
Updated -- just as an FYI, the more text transformation we apply on the README, the more likely it may break if styling on the markdown is significantly updated, but I've tried to keep it as flexible as possible.

image

**Comment by daltonmenezes** [comment]
@jfgordon2 Looks great for me, thanks. I'll keep the same style in that readme. Just a few fixes more. `uni bake how yarn-install`is not affected by the default recipe feature. Can you please fix it? After this, just update [this](https://github.com/uni-linux/uni/blob/master/README.md) and [this](https://github.com/uni-linux/uni/blob/master/docs/en/API.md) with your modifications and I can merge your pull request.

**Comment by jfgordon2** [comment]
@daltonmenezes updated for your review

**Review by daltonmenezes — APPROVED** [review]

**Comment by daltonmenezes** [comment]
@jfgordon2 Well done. Thank you so much. :D

**Pull request merged**